src/Types/index.ts

```diff
@@ -0,0 +1,22 @@
+export interface Post {
+  userId: number,
+  id: number,
+  title: string,
+  body: string,
+  user: User | null,
+  comments: Comments[],
+}
+
+export interface User {
+  id: number,
+  name: string,
+  email: string,
+}
+
+export interface Comments {
```

Comment:
```suggestion
export interface Comment {
```
Ти описуєш один коммент, а не багато комментів